geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -88,11 +89,27 @@ object AccumuloRunner extends Runner {
 
     val params = Option(command.params)
 
+    // TODO: handle OptionalCredentialsParams
```

Comment:
that's probably fine... if it's easy, might be safer to add handling for it in case it does get used some time in the future